### PR TITLE
D8CORE-2592

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Remove the duplicate ID's that happens on submenus for the mobile/responsive view.

# Review By (Date)
- Wednesday 10/21

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Create menu links to have sub menus on main pages.
3. change to the mobile view. 
4. Verify that that there are no more id='su-acs-

# Affected Projects or Products
- This is done on Stanford Basic and we also need it on SOE since that is where it was found.

# Associated Issues and/or People
- D8CORE-2592
- @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: dist/templates/decanter/components/global-footer/global-footer.twig [hunk 1]
```diff
@@ -11,38 +11,33 @@
...
 <div class="su-global-footer {{ modifier_class }}">
-  <div class="su-global-footer__container" title="Common Stanford resources">
+  <div class="su-global-footer__container">
     <div class="su-global-footer__brand">
       {%- include template_path_logo -%}
     </div>
     <div class="su-global-footer__content">
       <nav aria-label="global footer menu">
         <ul class="su-global-footer__menu su-global-footer__menu--global">
-          <li><a href="https://www.stanford.edu">Stanford Home{{ external_link_text }}</a></li>
```
Comment (pookmish) [inline dist/templates/decanter/components/global-footer/global-footer.twig:28]: Why is this and several other files being changed? they don't seem related to the topic of the PR

Comment (jenbreese) [reply dist/templates/decanter/components/global-footer/global-footer.twig:28]: I'm not sure why that happened. I thought it was a git sorting the files out thing. I pushed it up and then was in our team meeting. I'll look at it again.

Comment (pookmish) [reply dist/templates/decanter/components/global-footer/global-footer.twig:28]: You might just want to reset all the files that are unrelated. you can reset them by `git fetch; git checkout origin/8.x-1.x -- [filepath]`

Comment (jenbreese) [reply dist/templates/decanter/components/global-footer/global-footer.twig:28]: thanks!

**Comment by jenbreese** [comment]
I did something wrong so starting over.
